html/changelogs/Naelynn-PR-6902.yml

```diff
@@ -1,42 +1,41 @@
-################################

```

Comment:
Removes the example changelog

Reply:
when the fck was this done... I didn't mean to touch example changelog x_X

Reply:
It should be back now

Reply:
Nope that's not the example changelog you readded

Reply:
I genuinely have no idea where that came from. Should be fixed now. Thank you for checking!